unit_tests/utils/test_team_repo.py

```diff
@@ -1,10 +1,95 @@
 # -*- coding: utf-8 -*-
+import os
+
 from elitefurretai.utils.team_repo import TeamRepo
 
 
 def test_team_repo():
-    tr = TeamRepo("data/fixture")
-    assert "example_team" in tr.teams
+    tr = TeamRepo("data/teams")
+    assert "gen9vgc2024regg_speed_ability1" in tr.teams
+    assert "gen8vgc2020_s6_yan" in tr.teams
     assert len(tr.teams) > 5
     for team in tr.teams:
         assert isinstance(tr.teams[team], str)
+
+
+# helper function
```

Comment:
Just for hygiene and from personal preference, can you add the helper functions as the first functions in this file?